src/components/features/product-table.tsx

```diff
@@ -43,34 +52,68 @@ function ProductTable({
           <TableHeader>
             <TableRow>
               <TableHead className="w-[50px]">
-                <Checkbox onCheckedChange={check => handleChooseAllProduct(products, Boolean(check))} />
+                <Checkbox
+                  onCheckedChange={check =>
+                    handleChooseAllProduct(productVendors, Boolean(check))}
+                />
               </TableHead>
               <TableHead className="w-[200px]">Product Name</TableHead>
               <TableHead className="w-[200px]">Description</TableHead>
+              <TableHead className="w-[200px]">Price</TableHead>
               <TableHead>Last Modified</TableHead>
               <TableHead className="text-right"></TableHead>
             </TableRow>
           </TableHeader>
           <TableBody>
```

Comment:
So for snapshot testing, i think it would make much more sense to give the table `data-testid="some-table"` (change it to whatever you think the best name is)

And finally,
```ts
expect(screen.getByTestId('some-table').innerText).toMatchSnapshot()
```

This way, the snapshot content won't really contain the thing that's not being tested like Navbar, Footer, etc